### ISSUE TITLE
assets other than fonts are not being linked

### ISSUE THREAD

**Issue by danikenan** [facebook/react-native#14468]
Only font assets are collected and linked using `react-native link`. All other files are ignored.
```
"rnpm": {
  "assets": [
     "./someDir",
    "./anotherDir"
  ]
}

```
The other files can be in the same directory as the fonts or other directory.

**Comment by BaluRaut-oabstudios** [comment]
Can you give list of file names you are saying ignoring.

**Comment by hramos** [comment]
Hey, thanks for reporting this issue!

It looks like your description is missing some necessary information. Can you please add all the details specified in the [template](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md)? This is necessary for people to be able to understand and reproduce the issue being reported.

I am going to close this, but feel free to open a new issue with the additional information provided. Thanks!

**Comment by tslater** [comment]
@hramos It isn't clear that the link command only handles font files in terms of copying assets, but that is the case. I wanted to copy some xml files, but they were only copied when I changed the extension to ".otf" it copied. I finally found it in the source: https://github.com/facebook/react-native/blob/1673c570f984d86e88a3b6b44eb78f4848eb0515/local-cli/link/android/copyAssets.js#L18

I think that is misleading to call them assets...it should probably either be expanded to arbitrary files, or use the name "fonts" or "font files". Something like that. Should I follow the template and open a new issue?
